roles/pulp/tasks/main.yaml

```
@@ -10,44 +20,156 @@
     mode: "0755"
   loop: "{{ pulp_volumes }}"
 
+- name: Create Pulp storage subdirs
+  ansible.builtin.file:
+    path: "/var/lib/pulp/pulp_storage/{{ item }}"
```

```suggestion
    path: "/var/lib/pulp/{{ item }}"
```

And then you don't need the "Create Pulp temporary directory" task after this.
(You'll need `assets` once you try to upload stuff to pulp)